backend/test_context_fixes_demo.py

```
@@ -0,0 +1,461 @@
+"""
+Demo test showing context understanding and tool calling fixes working.
+
+This test demonstrates the fixes for:
+1. Understanding "those tasks" references
+2. Using tools instead of explaining
+3. Taking action instead of asking for clarification
+"""
+
+import asyncio
+import json
```

Ruff (0.12.2)

`json` imported but unused

Remove unused import: `json`

(F401)